packages/dataviews/src/components/dataviews-view-config/index.tsx

```diff
@@ -553,8 +556,7 @@ function _DataViewsViewConfig( {
 		<>
 			<ViewTypeMenu defaultLayouts={ defaultLayouts } />
 			<Dropdown
```

Comment:
I think it’d be a touch neater to move this inside `DataviewsViewConfigContent` and rename that to `DataviewsViewConfigDropdown`. Put another way, make a it self-contained component.

Reply:
Done https://github.com/WordPress/gutenberg/pull/65373/commits/c6afc268374e90dedc1b15c2ace991de3ec6ecbb